### PR TITLE
More updated documentation for SEDA (obsidian vault)

### PR DESCRIPTION
**Pull request by fiendskrah**
moved all of the UI markdown variable notation into `data/urban_institute/ui_markdown/`

SEDA3 summary docs in `data/SEDA/SEDA3_markdown/`

Current SEDA4 documentation as an obsidian vault in `data/SEDA/SEDA4_docs/` 
- Lost scientific notation and footnotes in this process. (pdf annotations > zotero > obsidian). This is hopefully in between a basic overview and an in-depth description. I am still working to make this prettier, and took some liberties in creating obsidian links to markdown files we might add later. Obsidian vaults shared through github seem like a very efficient way to flesh out the extent of papers that this project can produce.

### CODE REVIEW

**Review by knaaptime — CHANGES_REQUESTED** [review]
good start on this. A few things:

1. we definitely don't want to delete the `tools` directory, can you restore that?
2. can you rename the files so there are no spaces in the filenames
3. can you swap the zettelkasten syntax for generic markdown links? That way we dont lock into a specific software platform (obsidian) and the links will work directly when rendered on github

File: data/SEDA/SEDA3_markdown/SEDA_Overview.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
+# Stanford Education Data Archive (SEDA)
```
Comment (knaaptime) [inline data/SEDA/SEDA3_markdown/SEDA_Overview.md:1]: the formatting on this file makes it pretty hard to parse

**Comment by fiendskrah** [comment]
@knaaptime 
> good start on this. A few things:
> 
> 1. we definitely don't want to delete the `tools` directory, can you restore that?
> 2. can you rename the files so there are no spaces in the filenames
> 3. can you swap the zettelkasten syntax for generic markdown links? That way we dont lock into a specific software platform (obsidian) and the links will work directly when rendered on github

1. unsure what happened to tools/, this was unintentional. should be restored now.
2. spaces and parenthesis removed from file names - forgot how much of a problem this can cause.
3. I think this is resolved too, please verify that my changes to links conform to the correct syntax.

File: data/SEDA/SEDA3_markdown/SEDA_Overview.md (hunk 1 above)
Comment (fiendskrah) [reply data/SEDA/SEDA3_markdown/SEDA_Overview.md]: This was for SEDA 3 so a lot of the information is repeated. My recent push removes it - changes from SEDA 3 to SEDA 4 are [documented in section 4](https://github.com/spatialucr/geosnap4ed/pull/31/files#diff-0ccb09e37a7851370bba4266ede4f287efe14ae946a2488e62f0a503b7feb5c5).
